cmd/cartesi-rollups-cli/root/read/inputs/inputs.go

```diff
@@ -0,0 +1,53 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package inputs
+
+import (
+	"encoding/json"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/cartesi/rollups-node/internal/logger"
+	"github.com/cartesi/rollups-node/pkg/graphqlutil"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:     "inputs",
+	Short:   "Reads first N inputs from node GraphQL",
+	Example: examples,
+	Run:     run,
+}
+
+const examples = `# Read inputs from GraphQL:
+cartesi-rollups-cli read inputs --first 5 --graphqlEndpoint http://0.0.0.0:4000/graphql`
+
+var (
+	first           uint32
+	graphqlEndpoint string
+)
+
+func init() {
+	Cmd.Flags().Uint32Var(&first, "first", 0,
+		"index used to sign the transaction")
+
+	cobra.CheckErr(Cmd.MarkFlagRequired("first"))
+
+	Cmd.Flags().StringVar(&graphqlEndpoint, "graphql_endpoint", "http://0.0.0.0:4000/graphql",
+		"address used to connect to graphql")
+}
+
+func run(cmd *cobra.Command, args []string) {
+	logger.Init("info", true)
+
+	ctx := cmd.Context()
+	client := graphql.NewClient(graphqlEndpoint, nil)
+
+	resp, err := graphqlutil.GetInputs(ctx, client, int(first))
+	cobra.CheckErr(err)
+
+	val, err := json.MarshalIndent(resp.Inputs.Edges, "", "    ")
+	cobra.CheckErr(err)
+
+	logger.Info.Printf(string(val))
```

Comment:
I don't know whether we should use the logger here. If we do use the logger, it won't be possible to pipe the output to `jq`.

Reply:
This comment still applies